assets/js/components/ClustersList.jsx

```diff
@@ -71,7 +79,28 @@ function ClustersList() {
         filterFromParams: true,
         filter: (filter, key) => (element) =>
           element[key].some((sid) => filter.includes(sid)),
-        render: (_, { sid }) => sid.join(', '),
+        render: (_, { sid, id }) => {
+          const sidsArray = sid.map((singleSid, index) => [
+            index > 0 && ', ',
+            <SapSystemLink
+              key={`${id}-${singleSid}`}
+              systemType={get(
```

Comment:
Is not the lodash `get` an overkill?
`system?.type` would do the same.
Anyway, as this comes from other discussion, feel free to use it